modules/nodes-clusters-cgroups-2-install.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/clusters/nodes-cluster-cgroups-2.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-clusters-cgroups-2-install_{context}"]
+= Enabling Linux cgroup v2 during installation
+
+You can enable Linux control group version 2 (cgroup v2) when you install a cluster by creating installation manifests.
+
+.Procedure 
+
+. Edit the `config-node.yaml` manifest file to specify the `v2` cgroup. 
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Node
+metadata:
+annotations:
+include.release.openshift.io/ibm-cloud-managed: "true"
+include.release.openshift.io/self-managed-high-availability: "true"
+include.release.openshift.io/single-node-developer: "true"
+release.openshift.io/create-only: "true"
+name: cluster
+spec:
+cgroupMode: "v2"
```

Comment:
Is the formatting correct (indentations, mainly) for the 2 YAML blocks in this module?